dash.js

```diff
@@ -1140,8 +1140,15 @@ export const DockDash = GObject.registerClass({
  * @param actor
  */
 function ensureActorVisibleInScrollView(scrollView, actor) {
-    const {adjustment: vAdjustment} = scrollView.vscroll;
-    const {adjustment: hAdjustment} = scrollView.hscroll;
+    let vAdjustment = scrollView.vadjustment
```

Comment:
What about something like...?

`const vAdjustment = scrollView.vadjustment ?? scrollView.vscroll.adjustment`

Reply:
(and maybe add a comment before explaining that scrollView.vscroll/hscroll has been deprecated in Gnome Shell 46)

Reply:
ooh good point! i forgot that this operator exist in js :)

Reply:
I discovered it some months ago, and I really love it :-)

Reply:
updated it :+1: